File: Sheet 6 (Arrays 1)/D.py
t = 1
while True:
    n = int(input())
    if n == 0:
        break
    a = input()
    l = a.split()
    sm = 0
    for i in range(len(l)):
        l[i]= int(l[i])
        sm+= l[i]
    col = int (sm / n)
    ans = 0
    for i in range(len(l)):
        if l[i] > col:
           ans+= l[i] -col
    print("Set #" , t , sep = "")
    print("The minimum number of moves is " , ans , "." , sep ="")
    print()
    t+=1

File: Sheet 6 (Arrays 1)/D.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
    int n , t = 1;
    while (cin >> n && n){
            int sm = 0 ,arr[55];
    for (int i = 0 ; i < n ; i ++){
            cin >> arr[i];
    sm+=arr[i];
            }
            int ans = 0 , var = sm / n;
    for (int i = 0 ; i < n ; i ++){
            if (arr[i] > var)ans += arr[i] - var;
            }
          cout << "Set #" << t <<"\n";
          cout << "The minimum number of moves is " << ans <<".\n\n";
          t++;
    }
    return 0;
}
